modules/core/server/views/layout.server.view.html

```diff
@@ -56,7 +56,9 @@
 
   {% if livereload %}
   <!--Livereload script rendered -->
-  <script type="text/javascript" src="{{host}}:35729/livereload.js"></script>
+  <script type="text/javascript">
+    document.write('\x3Cscript type="text/javascript" src="//' + window.location.hostname + ':35729/livereload.js">\x3C/script>');
```

Comment:
Hehe, I didn't really like the code either, but was too lazy to really check it. Would this be better?
```
var scriptTag = document.createElement('script');
scriptTag.setAttribute('type', 'text/javascript');
scriptTag.setAttribute('src','//' + window.location.hostname + ':35729/livereload.js');
document.head.appendChild(scriptTag);